linux/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+services:
+  mmdvmhost:
+    build: https://github.com/f4hlv/MMDVMHost.git
```

Comment:
This is pointing to your github repo and not G4KLX's master - it may be more prudent for it to do so!

Reply:
Today the image is automatically created on the docker hub. As soon as there is a change on the github the image is updated. This value can be overridden if Dockerfile is present in MMDVMHost G4KLX's master.